src/cityreader/cityreader.py

```diff
@@ -1,11 +1,19 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
 
+class City():
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __str__(self):
+        return f"City({self.name}, {self.lat}, {self.lon})"
```

Comment:
Good use of this method to debug. Why did you choose to debug with this method instead of the `__repr__` method?